src/app/shared/components/header/header.component.ts

```diff
@@ -64,6 +53,26 @@ export class PLHMainHeaderComponent implements OnInit, OnDestroy {
       });
     }
   }
+
+  subscribeToAppConfigChanges() {
+    this.appConfigChanges$ = this.appConfigService.appConfig$.subscribe((appConfig: IAppConfig) => {
+      this.headerConfig = appConfig.APP_HEADER_DEFAULTS;
```

Comment:
nit(non-blocking) might be cleaner to put all the header config customisation into its own function.